CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 1.1.1   | [PR#379](https://github.com/BBC/psammead/pull/379) Updates for psammead-figure and psammead-paragraph |
```

Comment:
This PR doesn't change Psammead itself. We should avoid bumping the version of the root.